<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLangWindow</class>
 <widget class="QMainWindow" name="AddLangWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding language</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 218, 221)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 236, 236);
font: 63 15pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="LangLabel">
        <property name="styleSheet">
         <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Language</string>
        </property>
        <property name="buddy">
         <cstring>LangLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LangLineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="DeleteButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
